src/service/templates/models/rethinkdb/templates/service.model.js

```diff
@@ -0,0 +1,27 @@
+const path = require('path');
+const rethink = require('rethinkdbdash');
+
+module.exports = function (options) {
+
+  // Create a rethinkdb instance
+  const r = rethink({
+    db: options.db // @slajax todo: add servers to config
+  });
+
+  // <!--- This code can be removed!
+  // All it does is set up your DB table.
+  r.dbList().contains(options.db) // create db if not exists
```

Comment:
Get it. Makes sense. Can someone create an issue for it in https://github.com/feathersjs/feathers-rethinkdb? I want to fix up search and can get it in at the same time.